src/cmd-buildupload

```diff
@@ -72,8 +76,10 @@ def cmd_upload_s3(args):
         s3_copy(f'builds/{args.build}/{f}', bucket, f'{prefix}/{args.build}/{f}',
                 CACHE_MAX_AGE_METADATA, args.acl)
     if not args.skip_builds_json:
-        s3_copy('builds/builds.json', bucket, f'{prefix}/builds.json',
+        s3_copy(BUILDFILES['list'], bucket, f'{prefix}/builds.json',
                 CACHE_MAX_AGE_METADATA, args.acl, extra_args={}, dry_run=args.dry_run)
+        # And now update our cached copy to note we've successfully sync'd.
+        subprocess.check_call(['cp', '-pf', '--reflink=auto', BUILDFILES['list'], BUILDFILES['sourcedata']])
```

Comment:
This "unsyncs" the `sourceurl` though since there's no guarantee we're uploading to the same place we `buildprep`ed from. Should probably also just always overwrite it here?

Reply:
Fixed